taskchampion/src/replica.rs

```diff
@@ -201,6 +213,7 @@ impl Replica {
     /// Create a new, empty task with the given UUID.  This is useful for importing tasks, but
     /// otherwise should be avoided in favor of `new_task`.  If the task already exists, this
     /// does nothing and returns the existing task.
+    #[deprecated(since = "0.7.0", note = "please use BasicTask instead")]
```

Comment:
The deprecation seems a little premature since you can't import a `BasicTask` into a `Replica` yet because there isn't a mechanism to commit `Operations` into a `Replica` yet, right? (Feel free to let this slide if that's in a subsequent PR I haven't reviewed yet.)

Reply:
Never mind, I see that's literally the next PR.